beta/nncf/tensorflow/sparsity/magnitude/algorithm.py

```diff
@@ -74,17 +80,19 @@ def get_transformation_layout(self, model):
 
                 transformations.register(
                     TFInsertionCommand(
-                        target_point=TFLayerWeight(original_node_name, weight_def.weight_attr_name),
+                        target_point=TFLayerWeight(target_layer_name, weight_def.weight_attr_name),
                         callable_object=BinaryMask(op_name),
                         priority=TransformationPriority.SPARSIFICATION_PRIORITY
                     ))
 
+        # Currently only applicable for mobilenet_v2_slim. TODO: expand custom layers
+        # as part of building an NNCFGraph.
```

Comment:
Please, update TODO according to the Style Guide. I want to note, that it is applicable for all model which has custom layers.

Reply:
Done